src/main/scala/scalismo/mesh/MeshOperations.scala

```diff
@@ -116,8 +116,10 @@ class TriangleMesh3DOperations(private val mesh: TriangleMesh[_3D]) {
 
     def grad(pt: Point[_3D]) = {
       val closestPt = closestPoint(pt).point
-      val grad = EuclideanVector(pt(0) - closestPt(0), pt(1) - closestPt(1), pt(2) - closestPt(2))
-      grad * (1.0 / grad.norm)
+
+      val grad = closestPt - pt
```

Comment:
Doesn't that change the direction of the gradient? before it was pt - closestPoint no?

Reply:
nice catch. Thanks for pointing this out.